File: Solution/Java/AssessmentEntities/CandidateAnswer.java
// package com.transflower.tflAssessment.entities;

public class CandidateAnswer {
    
     private int id;     
        private int candidateId;
        private int testQuestionId;
        private String answerKey;

        public CandidateAnswer()
        {

        }
        public CandidateAnswer(int id, int candidateId,int testQuestionId,String answerKey)
        {  
            this.id=id;
            this.candidateId=candidateId;
            this.testQuestionId=testQuestionId;
            this.answerKey=answerKey;
        }
        public int getId()
        {
            return id;
        }
        public void setId(int id)
        {
            this.id=id;
        }
        public int getCandidateId()
        {
            return candidateId;
        }
        public void setCandidateId(int candidateId)
        {
            this.candidateId=candidateId;
        }
        public int getTestQuestionId()
        {
            return testQuestionId;
        }
        public void setTestQuestionId(int testQuestionId)
        {
            this.testQuestionId=testQuestionId;
        }
        public String getAnserKey()
        {
            return answerKey;
        }
        public void setAnswerKey(String answerKey)
        {
            this.answerKey=answerKey;
        }

}

File: Solution/WebSolution/Dotnet/AssessmentPortal/Monolithic/Monolithic_Controller/AssessmentEntities/CandidateAnswer.cs
namespace Transflower.TFLAssessment.Entities
{
    public class CandidateAnswer
    {
        public int Id { get; set; }          // Primary Key (not needed for insert, but good for completeness)
        public int CandidateId { get; set; } // Foreign Key 
       public int AssessmentId { get;set; }
        public int TestQuestionId { get; set; } // Foreign Key
        public string AnswerKey { get; set; } // Answer
    }
}
